build-logic/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle

```diff
@@ -589,8 +589,8 @@ performanceTest.registerTestProject("workerApiProject", Copy) {
 }
 
 tasks.withType(JvmProjectGeneratorTask).configureEach {
-    if (project.hasProperty("springDmPluginVersion")) {
-        templateArgs['springDmPluginVersion'] = springDmPluginVersion
+    if (providers.gradleProperty("springDmPluginVersion").isPresent()) {
+        templateArgs['springDmPluginVersion'] = providers.gradleProperty("springDmPluginVersion").get()
```

Comment:
💅  Maybe extract the property into a local variable?

Reply:
Done